video_xblock/tests/test_fields.py

```diff
@@ -0,0 +1,90 @@
+"""Tests for classes defined in fields.py."""
+
+import datetime
+import unittest
+
+from video_xblock.fields import RelativeTime
+
+
+class RelativeTimeTest(unittest.TestCase):
+    """
+    Test RelativeTime field.
+    """
+    delta = RelativeTime()
```

Comment:
It was decided, that python representation of start_time and end_time should be python datetime.timedelta object, to be consistent with common time representation.